src/cs/production/Flecs/Expression/Expression.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Copyright (c) Flecs Hub (https://github.com/flecs-hub). All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the Git repository root directory for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Flecs;
+
+public class Expression
+{
+    private World _world;
+
+    public Expression(World world)
+    {
+        _world = world;
+    }
+
+    public TermBuilder Term<T>()
+        where T : unmanaged, IFlecsComponent
+    {
+        var termBuilder = new TermBuilder(_world);
+        termBuilder.First<T>();
+        return termBuilder;
+    }
+
+    public TermBuilder Term<T1, T2>()
+      where T1 : unmanaged, IFlecsComponent
+      where T2 : unmanaged, IFlecsComponent
+    {
+        var termBuilder = new TermBuilder(_world);
+        termBuilder.First<T1>();
+        termBuilder.Second<T2>();
+        return termBuilder;
+    }
+
+    public TermBuilder Term()
+    {
+        return new TermBuilder(_world);
+    }
+
+    public class TermBuilder
```

Comment:
Can we have one class/struct per file?

Reply:
Of course

Reply:
But it does not make sense, does it?
The term builder is not something you would use outside that scope, other than for chaining

Reply:
Create a folder then for everything in related to expressions? I see you already did that.

Reply:
Alrighty